server/src/main/java/org/solarbank/server/service/CalculateService.java

```diff
@@ -17,26 +25,86 @@ public class CalculateService {
     public CalculateResult processCalculateRequest(
         PanelSize panelSize,
         Double panelEfficiency,
-        EnergyTariff energyTariff
+        EnergyTariff energyTariff,
+        Map<String, Double> nasaData
     ) {
+        Double panelArea = panelSize.getHeight() * panelSize.getWidth();
+
+        Table monthlyAverageTable = calculateEnergyGenPerMonth(nasaData, panelEfficiency, panelArea);
```

Comment:
calls the method that calculates monthlyAverage Solar radiation, returning the result as a Tablesaw Table object. Any ideas of an alternative to Tablesaw as I think its a bit shit